manager/controlapi/service.go

```diff
@@ -141,6 +141,18 @@ func validateContainerSpec(container *api.ContainerSpec) error {
 	return nil
 }
 
+func validatePluginSpec(plugin *api.PluginSpec) error {
+	if plugin.Image == "" {
+		return grpc.Errorf(codes.InvalidArgument, "PluginSpec: image reference must be provided")
+	}
+
+	if _, err := reference.ParseNormalizedNamed(plugin.Image); err != nil {
```

Comment:
For validation purposes, we might want to disallow digest-only references (if a type assertion to `reference.Canonical` succeeds and a type assertion to `reference.NamedTagged` does not).